packages/@react-stately/selection/src/SelectionManager.ts

```diff
@@ -91,6 +91,14 @@ export class SelectionManager implements MultipleSelectionManager {
       : this.state.selectedKeys;
   }
 
+  /**
+   * The raw selection value for the collection.
+   * Either 'all' for select all, or a set of keys.
+   */
+  get rawSelection(): ISelection {
```

Comment:
Is there a better name for this? Currently, `SelectionManager` always returns a set of keys from `selectedKeys` even when the selection is select all. For the announcements, we need to be able to distinguish between them.

Reply:
`selection`
`selectionState`
that's all i've got, i think rawSelection is ok though as well, it implies that selectedKeys does something to the underlying data